configurations/performance/cassandra_stress_gradual_load_reduced_steps_number.yaml

```diff
@@ -1,4 +1,8 @@
 # Define load ops for steps
-perf_gradual_threads: {"read": 620, "write": 400, "mixed": 1900}
+# # For debugging, you can set a specific thread count for each step (per load).
+## The value of perf_gradual_threads[load] must be either:
+##   - a single-element list (applied to all throttle steps), or
```

Comment:
I'd say we should have `int` for all steps and `list` for different threads per step

Reply:
@soyacz 
I am not sure I understand what do you mean

Reply:
I mean sct could handle a case when threads are int type - then apply to every step, and in case it's a list, assign threads accordingly